### ISSUE TITLE
Feature request: show name/artist(album?) of song on specified actions (Companion to #9)

### ISSUE THREAD

**Issue by eejd** [Doraemoe/Spotify-Plugin-for-Quicksilver#10]
For at least the action of saving a song, it would be nice to have Quicksilver show the song/artist/album or some subset when triggered. It would be nice if this would be true optionally when the action has 'Show Window' selected in the trigger preferences, if possible.

In parallel, an action showing the same information would be nice (i.e. identical to the 'Show Playing Track' trigger from the iTunes plugin).

The 'Set Status to Spotify Currently Playing' AppleScript suggest that this information is accessible.

**Comment by Doraemoe** [comment]
So I added a show playing track command in the newest version. You could check that out. Will consider adding notification for add/save action.

**Comment by Doraemoe** [comment]
Notification function added, please check the new version.
